lib/server.js

```diff
@@ -413,6 +437,43 @@ WalletService.prototype.joinWallet = function(opts, cb) {
 };
 
 /**
+ * Save copayer preferences for the current wallet/copayer pair.
+ * @param {Object} opts
+ * @param {string} opts.email - Email address for notifications.
+ */
+WalletService.prototype.savePreferences = function(opts, cb) {
+  var self = this;
+
+  opts = opts || {};
+
+  self._runLocked(cb, function(cb) {
+    var preferences = Model.Preferences.create({
+      walletId: self.walletId,
+      copayerId: self.copayerId,
+      email: opts.email,
```

Comment:
We should check `opts.email.length` < 254 , and `opts.email` is a valid email address.